### DISCUSSION TITLE
Transfer a project from one Hobby account to another

### DISCUSSION THREAD

**Discussion by m99coder** [vercel/community#4688] [Account Issue] [answered]
### Question

Hi lovely Vercel Team,

I’m looking for a way to transfer one of my projects to another Vercel Hobby account. One I’m not in a team with. So totally unassociated. The same as transferring a GitHub repository to just another user account.

All the [documentation](https://vercel.com/docs/projects/overview#transferring-a-project) and guides I read around this just mention:

> Select the Vercel account or team you wish to transfer the Project to.

In the modal where I select the new owner I can type anything, but don’t get any results when I fill in the username of the intended new owner.

![vercel-transfer-project-modal](https://github.com/vercel/community/assets/3276992/c11fa0ed-00de-46df-8ad2-3c7806c05ef0)

Any help is welcome. Thanks in advance.

All the best,
Marco

**Comment by yashraut362** [comment]
any updates here ?

**Reply by leerob** [reply, marked as answer]
If you want to transfer projects from one team to another, you'll need to be a member of both teams. Is that the case?

**Reply by m99coder** [reply]
As the topic owner, in my case there is no team on the target side, so to speak. Just an account.

Would it work like this?

1. Create a team on the target side
2. Become a member of that team
3. Transfer the project to that team
4. Resign from that team

All the best,
Marco

**Reply by leerob** [reply]
All Vercel accounts are part of a Hobby team (this was a recent change) https://vercel.com/changelog/2024-01-account-changes.

Your understanding sounds correct.

**Reply by m99coder** [reply]
Thank you @leerob for clarifying it 👍
